### DISCUSSION TITLE
blank screen withe an esp32 simple

### DISCUSSION THREAD

**Discussion by defaliz** [Bodmer/TFT_eSPI#2307] [Q&A - General]
I ,
i've an esp32 dev module (not S2 or S3)
no problem with this library https://github.com/Xinyuan-LilyGO/TTGO-T-Display
but when i want use bodmer tft_espi library i've allways a black sceen..
i just used the examples..
is there some modifications to do in file .h to it withe esp32 simple module ?
regards

**Comment by Bodmer** [comment]
The User_Setup_Select.h file in the libray folder must be modified to select one display.

**Reply by defaliz** [reply]
Thank a lot bodmer, for quick answer..
